cu29/src/config.rs

```diff
@@ -227,15 +232,23 @@ pub struct Cnx {
     dst: String,
 
     /// Message type exchanged betwee src and dst.
-    msg: String,
+    pub msg: String,
+
+    /// Tells Copper to batch messages before sending the buffer to the next node.
+    /// If None, Copper will just send 1 message at a time.
+    /// If Some(n), Copper will batch n messages before sending the buffer.
+    pub batch: Option<u32>,
+
+    /// Tells Copper if it needs to log the messages.
```

Comment:
Does 'log' entail both serialization and writing to disk here?

Reply:
yes.
Also we need to get to vocabulary right, I am so bad at this :).